snorkel/learning/utils.py

```diff
@@ -74,6 +74,51 @@ def score(self, test_marginals, train_marginals=None, b=0.5, set_unlabeled_as_ne
 
 class MentionScorer(Scorer):
     """Scorer for mention level assessment"""
+
+    def score_b(self, test_marginals, b, get_candidates=False, set_unlabeled_as_neg=True, set_at_thresh_as_neg=True):
```

Comment:
I don't really get why we need to separate into `score` and `score_b`?  Maybe it's just the name in particular that bugs me.  Maybe `score_b` -> `_score_buckets`?